### PR TITLE
Feature/172/use case extensibility

### LINKED ISSUE #172: [FEATURE] Use case extensibility
**Is your feature request related to a problem? Please describe.**
Currently, all ACE-Box use cases are integrated in the same code (repo) that also contains VM provisioning and configurations. In order to decouple ACE-Box configurations from use case configurations and therefore provide an easier way to extend or add new use cases, these capabilities shall be split.

### PR DESCRIPTION
**Pull request by tobigremmer-dt**
Resolves #172 #214

### CODE REVIEW

File: user-skel/ansible/roles/keptn/tasks/job-executor-service.yml [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+    api_version: v1
+    kind: Secret
+    state: present
+    namespace: "{{ keptn_job_executor_service_namespace }}"
+    resource_definition:
+      type: Opaque
+      data:
+        DT_TENANT: "{{ dynatrace_tenant_url | regex_replace('https://', '') | b64encode }}"
+        DT_API_TOKEN: "{{ dynatrace_api_token | b64encode }}" 
+        DT_PAAS_TOKEN: "{{ dynatrace_paas_token | b64encode }}"
```
Comment (kristofre) [inline user-skel/ansible/roles/keptn/tasks/job-executor-service.yml:32]: Should this not be done by the `keptn` cli?

File: user-skel/ansible/roles/keptn/tasks/main.yml [hunk 2]
```diff
@@ -52,22 +52,49 @@
           lookAndFeelUrl: "{{ keptn_look_and_feel_url }}"
         features:
           automaticProvisioningURL: "http://keptn-gitea-provisioner-service.{{ keptn_namespace }}"
+      continuous-delivery:
+        enabled: true
+        ingressConfig:
+          ingress_hostname_suffix: "acebox-tg.ace-innovation.info"
```
Comment (kristofre) [inline user-skel/ansible/roles/keptn/tasks/main.yml]: linked to your specific ingress

**Review by kristofre — CHANGES_REQUESTED** [review]
looks good, requested some minor changes

File: user-skel/ansible/roles/keptn/tasks/job-executor-service.yml (hunk 1 above)
Comment (tobigremmer-dt) [reply user-skel/ansible/roles/keptn/tasks/job-executor-service.yml:32]: Needs to be deployed in the Job executor namespace (which has to be different to the default Keptn namespace). Keptn CLI creates secrets in the default Keptn namespace.

**Review by kristofre — APPROVED** [review]

**Pull request merged**